### ISSUE TITLE
Probability Greater than one for Guassian Process

### ISSUE THREAD

**Issue by ablanch5** [tensorflow/probability#1099]
I'm trying to use a basic Gaussian Process in Tensorflow using tensorflow probability. I use tensorflow to create a Gaussian process, create a sample, and then calculate the probability of that sample using log_prob method. However the result I get is positive meaning the probability is greater than one and I don't know why. I'm running the following in google Colab

import tensorflow as tf
import numpy as np

time_points = tf.constant(np.arange(0, 1, 0.1).reshape(-1, 1).astype(np.float32))

kernel = tfp.math.psd_kernels.ExponentiatedQuadratic(1., 1.)
gp = tfd.GaussianProcess(kernel, time_points)
samples = gp.sample()
log_p = gp.log_prob(samples)
# log_p is positive meaning the probability is greater than 1.

**Comment by brianwa84** [comment]
Remember that for a continuous distribution, dist.prob computes the
probability *density*.

The simplest thing to study would be to look at tfd.Uniform(0,
.5).prob(.25), which is 2. A uniform density of 2 over an volume interval
of .5 integrates to a total mass of 1.

**Comment by ablanch5** [comment]
Right sorry I was thinking Gaussian process is discrete and therefore we could compute probability. I realize that is not the right forum but if I'm using it as a loss function I'm anticipating problems if I can get negative values for the loss or is that not a problem? I'll close this in 1 day either way to remove it as an issue.

**Comment by brianwa84** [comment]
You can use the negative log_prob as a loss. Negative log likelihood based
training is quite common, though there are much simpler distributions than
a GP to get started with.
